frontend/src/Components/Headers/Header.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import Note from "plaid-threads/Note";
+
+import Link from "../Link/Link";
+
+import styles from "./Header.module.scss";
+
+interface Props {
+  linkToken: string | null;
+  linkSuccess: boolean;
+  setLinkSuccess: (arg: boolean) => void;
+  setIsItemAccess: (arg: boolean) => void;
+  isItemAccess: boolean;
+  currentPath: string;
+}
+const Header = (props: Props) => {
+  const [itemId, setItemId] = useState("");
+  const [accessToken, setAccessToken] = useState("");
+
+  return (
+    <div className={styles.grid}>
+      {" "}
+      <h3 className={styles.title}>Plaid Quickstart</h3>
+      {!props.linkSuccess && (
+        <>
+          <h4 className={styles.subtitle}>
+            A sample end-to-end integration with Plaid
+          </h4>
+          <p className={styles.introPar}>
+            The Plaid flow begins when your user wants to connect their bank
+            account to your app. Simulate this by clicking the button below to
+            launch Link - the client-side component that your users will
+            interact with in order to link their accounts to Plaid and allow you
+            to access their accounts via the Plaid API.
+          </p>
+          {/* message if backend is not running and there is no link token */}
+          {props.linkToken == null && (
```

Comment:
so `linkToken` can be `"" | null | "actual-link-token..."` Could we use an error state instead? This becomes easier if we use react context as well.

Reply:
done  - added context